docs/configuration/email.md

```diff
@@ -0,0 +1,5 @@
+As of Zanata 3.6? (https://github.com/zanata/zanata-server/pull/633) email configuration is taken directly from JBoss/WildFly's standalone.xml configuration, using the mail session configured with the JNDI name "java:jboss/mail/Default".  In the default configuration of JBoss/WildFly, this expects an SMTP server on localhost port 25.
```

Comment:
Should this have a heading, or does it get a heading automatically from somewhere else and does not need it because it has only one section?